lib/parse-args.js

```diff
@@ -152,6 +152,11 @@ function buildYargs (withCommands = false) {
       describe: 'supplying --allowExternal will cause c8 to allow files from outside of your cwd. This applies both to ' +
         'files discovered in coverage temp files and also src files discovered if using the --all flag.'
     })
+    .options('project-root', {
+      default: undefined,
+      type: 'string',
+      describe: 'specifies which points point to the root of the mono repository'
```

Comment:
I wonder if this option overlaps with the undocumented `resolve` option. Have you tried using it as the `projectRoot` in istanbul-reporter options?

```js
projectRoot: this.resolve || process.cwd(),
```

With the current changes it feels like you'd need to keep `projectRoot` and `resolve` in sync manually.

https://github.com/bcoe/c8/blob/7f1069dcce6821a6794557b39d6a13f620c64bad/lib/parse-args.js#L136-L139

https://github.com/bcoe/c8/blob/7f1069dcce6821a6794557b39d6a13f620c64bad/lib/report.js#L97

Reply:
To be honest, I don't understand why the 'resolve' option is needed. Whatever I specify there, it does not affect the final files.

Reply:
I tried using this, but it didn't help me.

Reply:
projectRoot only affects paths in lcov files. We are currently using a fork in a working project and it works fine.

Reply:
I did some testing with the `this.resolve` option now, and it indeed seems to have no effect at all. The `v8ScriptCov.url` paths are absolute, not relative. No idea what that API is supposed to do.